docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md

```diff
@@ -0,0 +1,251 @@
+Contents
+=============
+
+   * [在 Kubernetes 上部署 RadonDB MySQL 集群](#在-kubernetes-上部署-radondb-mysql-集群)
+      * [简介](#简介)
+      * [部署准备](#部署准备)
+      * [部署步骤](#部署步骤)
+         * [步骤 1：克隆代码](#步骤-1-克隆代码)
+         * [步骤 2：部署 Operator](#步骤-2-部署-operator)
+         * [步骤 3：部署 RadonDB MySQL 集群](#步骤-3-部署-radondb-mysql-集群)
+      * [部署校验](#部署校验)
+         * [校验 RadonDB MySQL Operator](#校验-radondb-mysql-operator)
+         * [校验 RadonDB MySQL 集群](#校验-radondb-mysql-集群)
+      * [连接 RadonDB MySQL](#连接-radondb-mysql)
+         * [同 NameSpace 访问](#同-namespace-访问)
+         * [跨 NameSpace 访问](#跨-namespace-访问)
+      * [配置](#配置)
+         * [容器配置](#容器配置)
+         * [节点配置](#节点配置)
+         * [持久化配置](#持久化配置)
+      * [参考](#参考)
+
+# 在 Kubernetes 上部署 RadonDB MySQL 集群(Operator)
+
+## 简介
+
+RadonDB MySQL 是一款基于 MySQL 的开源、高可用、云原生集群解决方案。支持一主多从高可用架构，并具备安全、自动备份、监控告警、自动扩容等全套管理功能。目前已经在生产环境中大规模的使用，包含**银行、保险、传统大企业**等。
+
+RadonDB MySQL 支持在 Kubernetes 上安装部署和管理，自动执行与运行 RadonDB MySQL 集群有关的任务。
+
+本教程主要演示如何在 Kubernetes 上部署 RadonDB MySQL 集群(Operator)。
+
+## 部署准备
+
+* 已准备可用 Kubernetes 集群。
+
+## 部署步骤
+
+### 步骤 1: 克隆代码
+
+```github
+git clone https://github.com/radondb/radondb-mysql-kubernetes.git
+```
+
+### 步骤 2: 部署 Operator
+
+```helm
+<For Helm V3>
+helm install <release> radondb-mysql-kubernetes/charts/mysql-operator
+```
+
+以下指定 release 名为 `demo` , 创建一个名为 `demo-mysql-operator` 的 [Deployment](#7-deployments)，以及创建一个名为 `clusters.mysql.radondb.com` 的 [CRD](#8-CRD)。
+
+```helm
+<For Helm V3>
+helm install demo radondb-mysql-kubernetes/charts/mysql-operator
```

Comment:
add `helm list` after install success, like: 
```
hustjason:gravity gaoriyao$ helm list
NAME     	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART              	APP VERSION
demo     	default  	1       	2021-05-18 11:44:35.793793 +0800 CST	deployed	radondb-mysql-1.0.0	5.7.33     
testmysql	default  	1       	2021-05-18 09:37:38.548275 +0800 CST	deployed	mysql-8.5.8        	8.0.25     
```

Reply:
In this way, it seems a bit redundant, which one is better?